package.json

```diff
@@ -7,9 +7,8 @@
     "dev": "vite dev"
   },
   "dependencies": {
-    "instant-noodles": "^0.0.3",
-    "react": "<18.0.0",
-    "react-dom": "<18.0.0"
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
   },
   "devDependencies": {
     "@testing-library/react": "12.1.4",
```

Comment:
Hello @YilikaLoufoua , looks like we can remove the `@testing-library/react` as well too. Would you be able to add this to the updates?

Reply:
Oops, will fix that asap!

Reply:
No worries at all, thank you for adding :)